### ISSUE TITLE
error message trying to compile rust_binary on MacOS

### ISSUE THREAD

**Issue by ulfjack** [bazelbuild/rules_rust#16]
```
$ bazel build :example
INFO: Found 1 target...
ERROR: /Users/ulfjack/Projects/experiment/Rust/BUILD:3:1: error executing shell command: 'set -e; rm -rf bazel-out/local-fastbuild/bin/example.deps; mkdir bazel-out/local-fastbuild/bin/example.deps
 LD_LIBRARY_PATH=external/rust_darwin_x86_64/rustc/lib DYLD_LIBRARY_PATH=external/rust_da...' failed: bash failed: error executing command /bin/bash -c ... (remaining 1 argument(s) skipped): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 101.
error: could not exec the linker `external/local_config_cc/cc_wrapper.sh`: No such file or directory (os error 2)
Target //:example failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 2,896s, Critical Path: 0,49s
```

**Comment by ulfjack** [comment]
Output from bazel build -s :example:

```
>>>>> # //:example [action 'Compiling Rust binary example (1 files)']
(cd /private/var/tmp/_bazel_ulfjack/307797bec2f572b7de7094efab12d866/execroot/Rust && \
  exec env - \
    PATH=/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Users/ulfjack/Software/:/Users/ulfjack/AndroidSdk/platform-tools/:/opt/local/bin/ \
    TMPDIR=/var/folders/1r/__q2nysj2tsfs_3tmq9781h80000gn/T/ \
  /bin/bash -c 'set -e; rm -rf bazel-out/local-fastbuild/bin/example.deps; mkdir bazel-out/local-fastbuild/bin/example.deps
 LD_LIBRARY_PATH=external/rust_darwin_x86_64/rustc/lib DYLD_LIBRARY_PATH=external/rust_darwin_x86_64/rustc/lib external/rust_darwin_x86_64/rustc/bin/rustc example.rs --crate-name example --crate-type bin -C opt-level=3 --codegen ar=/usr/bin/ar --codegen linker=external/local_config_cc/cc_wrapper.sh --codegen link-args='\'''\'' -L all=external/rust_darwin_x86_64/rust-std-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib --out-dir bazel-out/local-fastbuild/bin --emit=dep-info,link')
```

**Comment by ulfjack** [comment]
It's missing a declared dependency on the C++ toolchain. I can work around the issue by adding a fake cc_library to my BUILD file:
```
cc_library(
    name = "foo",
    srcs = ["foo.cc"],
)
```

And then building both:
`bazel build :example :foo -k`

**Comment by ulfjack** [comment]
Note that this doesn't work at head due to https://github.com/bazelbuild/bazel/issues/1469

**Comment by ulfjack** [comment]
Actually, that bug isn't related after all.

**Comment by davidzchen** [comment]
Fixed by #19
